ports/ports.go

```diff
@@ -0,0 +1,27 @@
+package ports
+
+import (
+	"strconv"
+)
+
+// IsValid checks if a port is valid
+func IsValid(v interface{}) bool {
```

Comment:
we could use generic here :D Do we tied to older Golang version or we can use generics?